### PR TITLE
update proxy settings doc

### PR DESCRIPTION
**Pull request by mfanjie**
docs update for kubernetes/kubernetes#22414
addressed comments for #62  
@johndmulhausen could you please help to add labels on this PR, thank you~

### CODE REVIEW

File: docs/getting-started-guides/vagrant.md [hunk 1]
```diff
@@ -365,21 +365,25 @@ export KUBERNETES_NODE_MEMORY=2048
...
-Optionally you can specify addresses to not proxy, for example
+You can also specify addresses that bypass the proxy, for example:
 
-```sh
+```shell
 export KUBERNETES_NO_PROXY=127.0.0.1
 ```
 
-If you are using sudo to make kubernetes build for example make quick-release, you need run `sudo -E make quick-release` to pass the environment variables.
+If you are using sudo to make Kubernetes build, use the -E flag to pass in the environment variables. For example, if running make quick-release, use:
```
Comment (johndmulhausen) [inline docs/getting-started-guides/vagrant.md]: Wrap "-E" and "make quick-release" in backtics so they appear as code like this: `-E` and `make quick-release`

**Comment by johndmulhausen** [comment]
I just have one formatting note. Thanks for addressing the earlier comments. :)

**Comment by mfanjie** [comment]
@johndmulhausen any updates are required for this PR? the code for 22414 is LGTM, and the doc has been reviewed by @ixdy before, let me know if I need a update on this PR, please.

**Comment by johndmulhausen** [comment]
Thanks for pointing me to the old PR. You're all set.

**Pull request merged**